packages/server-renderer-feature-service/README.md

```diff
@@ -0,0 +1,28 @@
+# @feature-hub/server-renderer-feature-service
+
+[![Package Version](https://img.shields.io/npm/v/@feature-hub/server-renderer-feature-service.svg)](https://www.npmjs.com/package/@feature-hub/server-renderer-feature-service)
+
+A feature service to manage server-side-rendering.
```

Comment:
Maybe we need a disclaimer + link to the milestone for implementing the full functionality.

Reply:
Please remove the dash before rendering.